sources/Algo/include/Algo.h

```diff
@@ -393,5 +398,301 @@ class ControllerInterfaceDefinitionAlgorithm : public NodeAlgorithm {
  private:
   HvdcLineMap& hvdcLines_;  ///< the set of hvdc line, to know if we have already encountered this hvdc line
 };
+
+/**
+ * @brief DynModel definition
+ *
+ * Structure containing minimum information to define a dynamic model with its connections
+ */
+struct DynModelDefinition {
+  using DynModelId = std::string;  ///< alias for dynamic model id
+
+  /**
+   * @brief Macro connection definition
+   */
+  struct MacroConnection {
+    using MacroId = std::string;    ///< alias for macro connector id
+    using ElementId = std::string;  ///< alias for connected element id
+
+    /// @brief Connected element type
+    enum class ElementType {
+      NODE = 0,  ///< node type
```

Comment:
Node type, with a capital letter like for the other types

Reply:
OK